uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -1038,5 +1039,25 @@ private function deleteDefaultWorkPattern() {
     $workPatternTable = WorkPattern::getTableName();
     CRM_Core_DAO::executeQuery("DELETE FROM {$workPatternTable} WHERE is_default = 1");
   }
+
+  public function testGetBalanceChangeForToilLeaveRequest() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request' => true,
+    ]);
```

Comment:
It calls calculateToilExpiryDate only if the expiry_date is null (it's something I've changed recently in here https://github.com/civicrm/civihr/commit/e59c258c2bbbee75dbbe52cca39523525ed79777), which is not the case on the tests